hi-ml-azure/src/health_azure/himl.py

```diff
@@ -399,6 +525,7 @@ def submit_to_azure_if_needed(  # type: ignore
         ignored_folders: Optional[List[PathOrString]] = None,
         default_datastore: str = "",
         input_datasets: Optional[List[StrOrDatasetConfig]] = None,
+        v2_input_datasets: Optional[Dict[str, Input]] = None,
```

Comment:
I'm hesitant to change the format of the function names because it can be a pain when you're searching for a function if there's another with a matching name + suffix...